examples/default/main.c

```diff
@@ -47,6 +52,14 @@ int main(void)
     rtc_init();
 #endif
 
+#ifdef MODULE_GNRC
```

Comment:
Probably yes, but for now (before the release) I would just use the gnrc lower layer functionalities.